1-JavaScript/Week3/homework/ex6-totalCost.js

```diff
@@ -19,14 +19,25 @@ instead!
 3. Call the function once, giving it the object `cartForParty` as an argument.
    Use `console.log` to display the result.
 -----------------------------------------------------------------------------*/
-const cartForParty = {
-  // TODO complete this function
+const cartForParty = {// TODO complete this function
+  food: 40.30,
+  fruits: 17.70,
+  drinks: 30.50,
+  snacks: 22,
+  nuts: 18
 };
 
-function calculateTotalPrice(/* TODO parameter(s) go here */) {
-  // TODO replace this comment with your code
+function calculateTotalPrice(cart) {// TODO replace this comment with your code
```

Comment:
perfect!

It is also possible to solve this problem following a functional programming approach: 

```
function calculateTotalPrice(cart) {
  const amount = Object.values(object).reduce((totalPrice, price) => totalPrice + price, 0);
  return `Total: €${amount}`;
}
```

As you can see in this example, we can use the method Object.values to get an array with all the property values from an object:

```
console.log(Object.values(object)) // [40.3, 17.7, 30.5, 22, 18]
```

After getting them, we can call the method Array.reduce to calculate the total price.